DocSum/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -0,0 +1,112 @@
+## 🚀 Start Microservices and MegaService
```

Comment:
## -> #  will resolve this https://github.com/opea-project/GenAIExamples/actions/runs/11815151412/job/32932891261?pr=1125

Reply:
Ok, thank you for the tip.